bika/lims/databasesanitize/templates/main_template.pt

```diff
@@ -0,0 +1,28 @@
+<form method="post"
+      action="sanitize_action">
+
+    <div id="progress_bar">
+        <progress value="0" max="100"></progress>
+    </div>
+
+    <div>
+        <p> Select the tasks to carry on: </p>
+            <p>
+                <label>
+                    <input type="checkbox" name="analysis_date"
+                           value="1">
```

Comment:
To make the content translatable, please remember `i8n:translate` param.

Reply:
I'd also use a more explicit name than "analysis_date" for this task, such as "sanitize_analyses_creation_date". Note that if you do this change you'll need to replace it in the .py too.